src/openrct2/ride/Vehicle.cpp

```diff
@@ -6393,26 +6396,26 @@ int32_t Vehicle::UpdateMotionDodgems()
  *
  *  rct2: 0x006DD365
  */
-bool vehicle_update_dodgems_collision(Vehicle* vehicle, int16_t x, int16_t y, uint16_t* spriteId)
+bool Vehicle::UpdateDodgemsCollision(int16_t curX, int16_t curY, uint16_t* spriteId)
 {
-    uint16_t bp = (vehicle->var_44 * 30) >> 9;
-    uint32_t trackType = vehicle->track_type >> 2;
+    uint16_t bp = (var_44 * 30) >> 9;
```

Comment:
bp is the dodgems vehicles radius. Could you rename the variable to something like that.